big-practice/src/js/helpers/show-message.helper.js

```diff
@@ -0,0 +1,17 @@
+class DocumentHelper {
```

Comment:
Should use static class for Helper/Utilities

Reply:
updated!